tekton/src/pipelines/taskdefs/cluster-setup/ocp-verify.yml.j2

```diff
@@ -17,3 +17,5 @@
       value: {{ verify_workloads | default("True") }}
     - name: verify_ingress
       value: {{ verify_ingress | default("True") }}
+    - name: ocp_ingress_tls_secret_name
```

Comment:
Shouldn't this have a default value of "", rather than being a required parameter?

Reply:
Hmmn.. I thought that since this is coming from `params`, the default to "" would be coming from here? 
https://github.com/ibm-mas/cli/blob/andrercm-multiple-fixes/tekton/src/params/install.yml.j2#L4

Reply:
<img width="603" alt="image" src="https://github.com/ibm-mas/cli/assets/31037381/b77a4931-4090-477b-8d34-014b5e5eda7c">